app/Http/Controllers/HomeController.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+
+class HomeController extends Controller
+{
+    public function index()
+    {
+        return view('welcome');
+    }
+
+    public function show($id)
+    {
+        return "USER: ".$id;
+    }
+
+    public function ageChecker()
+    {
+        return "Super Secret Stuff";
+    }
+    
+    public function testPost(Route $route)
```

Comment:
Hajde da pratimo konvenciju, bice nam kasnije mnogo lakse za debagovanje ako je post ruta onda nam je store kontroler. Actions Handled By Resource Controller u laraveoj dokumentaciji mozemo videti koje akcije podudatju koje rute https://laravel.com/docs/5.8/controllers#resource-controllers. Ako je neki kontroler kao ageChecker to je ok da bude drugacije ime od CRUD operacija.